app/controllers/reviews_controller.rb

```diff
@@ -10,6 +10,7 @@ def create
     work = Work.find(params[:work_id])
     authorize! work, to: :review?
     if params[:state] == 'approve'
+      work.begin_deposit!
```

Comment:
I see we call this above each invocation of `DepositJob.perform_later(work)`, but I assume we *don't* want to move this code to the job because we want the state change to happen immediately. True?

Reply:
Yes, it should be set to the new state even while the job is enqueue.